grails-app/controllers/org/bbop/apollo/OrganismController.groovy

```diff
@@ -929,20 +929,20 @@ class OrganismController {
     ])
     @Transactional
     def updateOrganismInfo() {
-        log.debug "updating organism info ${params}"
+        println"updating organism info ${params}"
         try {
             JSONObject organismJson = permissionService.handleInput(request, params)
             permissionService.checkPermissions(organismJson, PermissionEnum.ADMINISTRATE)
             Organism organism = Organism.findById(organismJson.id)
             if (organism) {
                 log.debug "Updating organism info ${organismJson as JSON}"
                 organism.commonName = organismJson.name
-                organism.blatdb = organismJson.blatdb
-                organism.species = organismJson.species
-                organism.genus = organismJson.genus
-                organism.metadata = organismJson.metadata
+                organism.blatdb = organismJson.blatdb ?: null
```

Comment:
just to clarify, you are casting an empty string to null, right?

Reply:
Yes.